lib/dlss/capistrano/tasks/control_master.rake

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'open3'
+
+CONTROLMASTER_HOST = ENV.fetch('CONTROLMASTER_HOST', 'dlss-jump')
+CONTROLMASTER_SOCKET = ENV.fetch('CONTROLMASTER_SOCKET', "~/.ssh/%r@%h:%p")
```

Comment:
If you use a differently named `ProxyJump` host or `ControlPath` value in your SSH config, you can set some ENV variables in your shell to account for that. This way we can have some reasonable SSH config defaults documented in the [DeveloperPlaybook](https://github.com/sul-dlss/DeveloperPlaybook/blob/main/best-practices/ssh_configuration.md) but no one's obligated to use those defaults because we have a seam here for overriding what values this task will use.

Reply:
This probably means we need a corresponding README change. I'd welcome thoughts on what folks would expect that to look like.